src/eduid_userdb/db.py

```diff
@@ -286,7 +286,7 @@ def _get_documents_by_filter(
         skip: Optional[int] = None,
         limit: Optional[int] = None,
         raise_on_missing: bool = True,
-    ) -> List[Mapping]:
+    ) -> List[Dict]:
```

Comment:
I would think this change will be a problem for everything that expects a Mapping from _get_document_by_attr.

Reply:
:+1: